### ISSUE TITLE
No longer shipping types

### ISSUE THREAD

**Issue by SimonSchick** [rmariuzzo/react-new-window#45]
The `src` folder is no longer inside of the package, therefore

```
  "types": "./src/NewWindow.d.ts",
```

fails to be resolved.

**Comment by andrewsantarin** [comment]
@SimonSchick I know that workarounds aren't good, but until we get a fix for this, at least with [`create-react-app`](https://github.com/facebook/create-react-app), the simplest way I got this going was to download that file into my project's `./src` folder.

```sh
my-create-react-app-project
├── node_modules
├── src
│   ├── App.css
│   ├── App.test.tsx
│   ├── App.tsx
│   ├── index.css
│   ├── index.tsx
│   ├── ... # other files
│   └── react-new-window.d.ts # store the definition here.
├── ... # other files
└── README.md
```

Does that help?

**Comment by SimonSchick** [comment]
I know how to fix it, I am reporting it as broken by default.

**Comment by rmariuzzo** [comment]
Sorry for the late reply. I'm gonna fix that soon this week. If there's a
PR I could merge it quickly. I apologize.

**Comment by gianpaj** [comment]
@rmariuzzo looks like this PR fixes TS types https://github.com/rmariuzzo/react-new-window/pull/44

cc @jakubkottnauer

**Comment by jakubkottnauer** [comment]
@gianpaj indeed it does. Please take a look @rmariuzzo when you have time

**Comment by rmariuzzo** [comment]
Thank you all for your patience.

**Comment by gianpaj** [comment]
I believe 0.1.2 was released with #44 merged.
This issue can be closed
